scripts/apothecary/apothecary

```diff
@@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
 # paths to android SDK, etc
```

Comment:
Eventually to remove all the binaries from the repo and history.

On Thursday, 23 June 2016, Jonas Jongejan <notifications@github.com> wrote:

> In scripts/apothecary/apothecary
> <https://github.com/openframeworks/openFrameworks/pull/5059#discussion_r68139700>
> :
>
> > @@ -88,7 +88,8 @@ VS_64_BIT_ENV='VC\bin\x86_amd64\vcvarsx86_amd64.bat'
> >  # paths to android SDK, etc
>
> Is the idea to remove the formulas from main repo, but keep updating the
> .a files with new builds? Or is it to remove the .a files entirely from git
> soon, and depend on external download or manual build?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openframeworks/openFrameworks/pull/5059/files/23dd4480d0d93430a6dc9e3591df3906293ecc71..6f53cd61fb363bf6daa079d7d992d2ea6fec6ec0#r68139700>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAytHDnbnGkwWH8xAUKl3U3ElikfERBFks5qOaqIgaJpZM4II_Tu>
> .
>